src/components/Exercises/Detail/OverviewCard.tsx

```diff
@@ -15,44 +15,42 @@ type OverviewCardProps = {
 
 export const OverviewCard = ({ exerciseBase, language }: OverviewCardProps) => {
 
-
+    const navigate = useNavigate();
     const exercise = language ? exerciseBase.getTranslation(language) : exerciseBase.getTranslation(new Language(ENGLISH_LANGUAGE_ID, 'en', 'English'));
-    const [t] = useTranslation();
+    const [t] = useTranslation();    
 
     return (
-        <Link style={{textDecoration: "none"}} to={`${exerciseBase.id}`}>
-        <Card key={exerciseBase.id}>
-            <CardActionArea>
-                <CardMedia
-                    component="img"
-                    image="https://mui.com/static/images/cards/contemplative-reptile.jpg"
-                    alt="green iguana"
-                />
-                <CardContent>
-                    <Typography gutterBottom variant="h6" component="div">
-                        {exercise.nameLong}
-                    </Typography>
-
-                    <Chip
-                        label={t(getTranslationKey(exerciseBase.category.name))}
-                        key={exerciseBase.category.id}
-                        sx={{ position: "absolute", top: 8, left: 8 }}
-                        color="warning"
-                        size="small"
+            <Card key={exerciseBase.id} onClick={() => navigate(`../${exerciseBase.id}`)}>
```

Comment:
I haven't tested this, but wouldn't this make the links on the overview break? can we use absolute paths in `to`?